File: src/main/java/ac/dnd/server/admission/domain/event/ApplicantQueryEvent.java
package ac.dnd.server.admission.domain.event;

import ac.dnd.server.enums.ApplicantStatus;
import ac.dnd.server.enums.ApplicantType;

public record ApplicantQueryEvent(
	Long applicantId,
	String name,
	String email,
	ApplicantType type,
	ApplicantStatus status
) {
}

File: src/main/kotlin/ac/dnd/server/admission/domain/event/ApplicantQueryEvent.kt
package ac.dnd.server.admission.domain.event

import ac.dnd.server.admission.domain.enums.ApplicantStatus
import ac.dnd.server.admission.domain.enums.ApplicantType

data class ApplicantQueryEvent(
    val applicantId: Long,
    val name: String,
    val email: String,
    val type: ApplicantType,
    val status: ApplicantStatus
)
